CMakeLists.txt

```diff
@@ -554,7 +556,9 @@ add_jar(
     tests-code-jar
     SOURCES
         tst/com/amazon/corretto/crypto/provider/test/AccessibleByteArrayOutputStreamTest.java
+        tst/com/amazon/corretto/crypto/provider/test/AesGcmKatTest.java
```

Comment:
TODO [childw]: do full comparison of available tests against this list in CMakeLists.txt, ideally we'll just list the relevant directories to build this list automatically.

Reply:
ok, we're all set here. the only tests not included in this list are the benchmarks:

```
$ find tst -name *.java  | while read file; do grep $file CMakeLists.txt &>/dev/null || echo "NO ENTRY FOR $file"; done
NO ENTRY FOR tst/com/amazon/corretto/crypto/provider/test/SmallHashBench.java
NO ENTRY FOR tst/com/amazon/corretto/crypto/provider/test/SecureRandomBench.java
NO ENTRY FOR tst/com/amazon/corretto/crypto/provider/test/HMACBench.java
```

Reply:
Opened a tracking issue: https://github.com/corretto/amazon-corretto-crypto-provider/issues/239